formfyxer/data/gendered_terms.yml

```diff
@@ -0,0 +1,318 @@
+the right man for the job: the right one for the job
+gentlewomen's agreement: unwritten agreement
+gentlemen's agreement: unwritten agreement
+the man in the street: the person in the street
+woman of distinction: person of distinction
+no man is an island: no one is an island
+ladies & gentlemen: everyone
+grandfather clause: pre-existing clause
+grandmother clause: pre-existing clause
+man of distinction: person of distinction
+preferred pronouns: personal pronouns
+masculine pronouns: he/him pronouns
+prodigal daughter: prodigal child
+woman among women: outstanding person
+woman of the hour: person of the hour
+feminine pronouns: she/her pronouns
+confidence-woman: confidence-person
+founding fathers: founders
+founding mothers: founders
+woman about town: bon vivant
```

Comment:
I like the phrase, but it doesn't seem plain language. Maybe "popular person"?

Reply:
Another phrase we can probably leave in for now. We didn't make the word list so we're probably not in the best position to judge individual entries other than for "will the substitution work programmatically and does it match the intent of the authors". I also don't imagine this phrase will appear on real-world court forms.

Reply:
You know what, I'll just take this one out. Doesn't seem very relevant anyway. Good catch